### ISSUE TITLE
Mislabeled RTK edition

### ISSUE THREAD

**Issue by PANICBUTT0N** [Kuuuube/kanjigrid#28]
The RTK 3rd edition list currently in the addon is from either the 2nd or 1st edition of RTK3, as it has 830 kanji rather than the 800 RTK3 was reduced to in its 3rd edition.
Here are the 800 Kanji in RTK3 3rd edition (sourced from [koohii](https://kanji.koohii.com/)):
此柴些砦髭禽檎憐燐麟鱗奄庵掩悛駿峻竣犀皐畷綴鎧凱呑韮籤懺芻雛趨尤厖或兎也巴疋菫曼云莫而倭侠倦俄佃仔仇伽儲僑倶侃偲侭脩倅做冴凋凌凛凧凪夙鳳剽劉剃厭雁贋厨仄哨咎囁喋嘩噂咳喧叩嘘啄吠吊噛叶吻吃噺噌邑呆喰埴坤壕垢坦埠堰堵嬰姦婢婉娼妓娃姪嬬姥姑姐嬉孕孜宥寓宏牢宋宍屠屁屑屡屍屏嵩崚嶺嵌帖幡幟庖廓庇鷹庄廟彊弛粥挽撞扮捏掴捺掻撰揃捌按播揖托捧撚挺擾撫撒擢摺捷抉怯惟惚怜惇恰恢悌澪洸滉漱洲洵滲洒沐泪渾涜梁澱洛汝漉瀕濠溌湊淋浩汀鴻潅溢湛淳渥灘汲瀞溜渕沌濾濡淀涅斧爺猾猥狡狸狼狽狗狐狛獅狒莨茉莉苺萩藝薙蓑苔蕩蔓蓮芙蓉蘭芦薯菖蕉蕎蕗茄蔭蓬芥萌葡萄蘇蕃苓菰蒙茅芭苅葱葵葺蕊茸蒔芹苫蒼藁蕪藷薮蒜蕨蔚茜莞蒐菅葦迪辿這迂遁逢遥遼逼迄逗鄭隕隈憑惹悉忽惣愈恕昴晋晟暈暉旱晏晨晒晃曝曙昂昏晦膿腑胱胚肛脆肋腔肱胡楓楊椋榛櫛槌樵梯柑杭柊柚椀栂柾榊樫槙楢橘桧棲栖桔杜杷梶杵杖樽櫓橿杓李棉楯榎樺槍柘梱枇樋橇槃栞椰檀樗槻椙彬桶楕樒毬燿燎炬焚灸煽煤煉燦灼烙焔烹牽牝牡琳琉瑳琢珊瑚瑞玖瑛玲畢畦痒痰疹痔癌痺眸眩雉矩磐碇碧硯砥碗碍碩磯砺碓禦祷祐祇祢禄禎秤黍禿稔稗穣稜稀穆窺窄穿竃竪颯站靖妾衿袷袴襖笙筏簾箪竿箆箔笥箭筑篠纂竺箕笈篇筈簸粕糟糊籾糠糞粟繋綸絨絆緋綜紐紘纏絢繍紬綺綾絃縞綬紗舵聯聡聘耽耶蚤蟹蛋蟄蝿蟻蝋蝦蛸螺蝉蛙蛾蛤蛭蛎罫袈裟截哉詢諄讐諌諒讃訊訣詫誼謬訝諺誹謂諜註譬轟輔輻輯豹賎貰賑贖躓蹄蹟跨跪醤醍醐醇麹釦銚鋤鋸錐鍬鋲錫錨釘鑓鋒鎚鉦錆鍾鋏閃悶閤雫霞翰斡鞍鞭鞘鞄靭鞠顛穎頗頌頚餐饗蝕飴駕騨馳騙馴駁駈驢鰻鯛鰯鱒鮭鮪鮎鯵鱈鯖鮫鰹鰍鰐鮒鮨鰭鴎鵬鸚鵡鵜鷺鷲鴨鳶梟塵麒舅鼠鑿艘瞑暝坐朔曳洩彗慧爾嘉兇兜靄劫歎輿歪翠黛鼎鹵鹸虔燕嘗殆牌覗齟齬秦雀隼耀夷嚢暢廻欣毅斯匙匡肇麿叢肴斐卿翫於套叛尖壷叡酋鴬赫臥甥瓢琵琶叉乖畠圃丞亮胤疏膏魁馨牒瞥睾巫敦奎翔皓黎赳已棘祟甦剪躾夥鼾陀粁糎粍噸哩浬吋呎梵薩菩唖牟迦珈琲檜轡淵伍什萬邁燭逞燈裡薗鋪嶋峯埜龍寵聾慾嶽國脛勁祀祓躇壽躊饅嘔鼈

**Comment by Kuuuube** [comment]
Do you have a properly linkable source for this? I found a reddit post that agrees with this data but I can't find anything else.

Interestingly the data in kanjigrid doesn't even match the source it is said to come from... this is one of the things that was here before I took over maintaining the addon.

Here's the diff between what you're suggesting and what is currently in kanjigrid:
```diff
+乖
+剽
+捏
+訝
+贖
+隕
+靄
+齟
+齬
-亙
-亨
-伶
-佑
-佼
-侑
-俣
-倖
-傭
-僻
-卜
-喬
-孟
-尭
-峨
-嵯
-巌
-巽
-彪
-掟
-掠
-撹
-梧
-欝
-欽
-煕
-熔
-珪
-瑶
-甫
-籠
-聚
-舘
-舜
-詑
-諏
-躯
-郁
-鏑
```

**Comment by PANICBUTT0N** [comment]
> Do you have a properly linkable source for this? I found a reddit post that agrees with this data but I can't find anything else.
> 
> 
> 
> Interestingly the data in kanjigrid doesn't even match the source it is said to come from... this is one of the things that was here before I took over maintaining the addon.
> 
> 
> 
> Here's the diff between what you're suggesting (+) and what is currently in kanjigrid (-):
> 
> ```diff
> 
> +乖
> 
> +剽
> 
> +捏
> 
> +訝
> 
> +贖
> 
> +隕
> 
> +靄
> 
> +齟
> 
> +齬
> 
> -亙
> 
> -亨
> 
> -伶
> 
> -佑
> 
> -佼
> 
> -侑
> 
> -俣
> 
> -倖
> 
> -傭
> 
> -僻
> 
> -卜
> 
> -喬
> 
> -孟
> 
> -尭
> 
> -峨
> 
> -嵯
> 
> -巌
> 
> -巽
> 
> -彪
> 
> -掟
> 
> -掠
> 
> -撹
> 
> -梧
> 
> -欝
> 
> -欽
> 
> -煕
> 
> -熔
> 
> -珪
> 
> -瑶
> 
> -甫
> 
> -籠
> 
> -聚
> 
> -舘
> 
> -舜
> 
> -詑
> 
> -諏
> 
> -躯
> 
> -郁
> 
> -鏑
> 
> ```

Unfortunately no, I have looked far and wide for more info on RTK3 3rd edition but there seems to be significantly less documentation on it compared to its predecessor, I suppose it was a minor enough change for most people to not really care. I'm confident that there are sufficient resources to confirm that the addon's current list is from the 2nd edition, but as for getting the exact list of Kanji from the 3rd, I'm not so sure (although I do trust Koohii fairly well). One thing I do know for certain, however, is that there are 800 kanji in the 3rd edition, [as stated on the publisher's website](https://uhpress.hawaii.edu/title/remembering-the-kanji-3-writing-and-reading-the-japanese-characters-for-upper-level-proficiency/).

**Comment by Kuuuube** [comment]
Fixed with 5085f2364deeea35f76537e251e5504828fe66ed. I have to link the reddit post here instead of koohii but they both have the same data anyways so it's fine. I'll push a release but ankiweb is dying at the moment so it won't be available for awhile to auto update.
